Index: BusBookTicket.Customer/obj/rider.project.model.nuget.info
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>16965876703106031
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/BusBookTicket.Customer/obj/rider.project.model.nuget.info b/BusBookTicket.Customer/obj/rider.project.model.nuget.info
--- a/BusBookTicket.Customer/obj/rider.project.model.nuget.info	(revision 2ea7d324ddc900211d3279a25fa3fb99706930b7)
+++ b/BusBookTicket.Customer/obj/rider.project.model.nuget.info	(date 1696663361851)
@@ -1,1 +1,1 @@
-16965876703106031
\ No newline at end of file
+16966633546693554
\ No newline at end of file
Index: BusBookTicket/obj/rider.project.model.nuget.info
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>16965876716282507
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/BusBookTicket/obj/rider.project.model.nuget.info b/BusBookTicket/obj/rider.project.model.nuget.info
--- a/BusBookTicket/obj/rider.project.model.nuget.info	(revision 2ea7d324ddc900211d3279a25fa3fb99706930b7)
+++ b/BusBookTicket/obj/rider.project.model.nuget.info	(date 1696663361843)
@@ -1,1 +1,1 @@
-16965876716282507
\ No newline at end of file
+16966633546873104
\ No newline at end of file
