Index: BusBookTicket.Customer/obj/rider.project.model.nuget.info
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>16966626762251494
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/BusBookTicket.Customer/obj/rider.project.model.nuget.info b/BusBookTicket.Customer/obj/rider.project.model.nuget.info
--- a/BusBookTicket.Customer/obj/rider.project.model.nuget.info	(revision 428cad91ccdabd3a8cda56b26f81391f623b03da)
+++ b/BusBookTicket.Customer/obj/rider.project.model.nuget.info	(date 1696663404700)
@@ -1,1 +1,1 @@
-16966626762251494
\ No newline at end of file
+16966633923462000
\ No newline at end of file
Index: BusBookTicket/obj/rider.project.model.nuget.info
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>16965876716282507
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/BusBookTicket/obj/rider.project.model.nuget.info b/BusBookTicket/obj/rider.project.model.nuget.info
--- a/BusBookTicket/obj/rider.project.model.nuget.info	(revision 428cad91ccdabd3a8cda56b26f81391f623b03da)
+++ b/BusBookTicket/obj/rider.project.model.nuget.info	(date 1696663404709)
@@ -1,1 +1,1 @@
-16965876716282507
\ No newline at end of file
+16966633923666578
\ No newline at end of file
